tox.ini

```diff
@@ -18,6 +18,7 @@ passenv =
     HOME
     TEST_AGENTS
     LXD_DIR
+# FIXME would it be easier to `pip install -e .`?
```

Comment:
to avoid redundant dependency specification? It does seem like there should be a better way than this

Reply:
Please see a separate PR, #1205